tests/chordsymbol_tests.ts

```diff
@@ -39,18 +38,11 @@ const note = (factory: Factory, keys: string[], duration: string, chordSymbol: C
   factory.StaveNote({ keys, duration }).addModifier(chordSymbol, 0);
 
 /** Calculate the glyph's width in the current music font. */
-// How is this different from Glyph.getWidth()? The numbers don't match up.
-function getGlyphWidth(glyphName: string): number {
-  // `38` seems to be the `fontScale` specified in many classes, such as
-  // Accidental, Articulation, Ornament, Strokes. Does this mean `38pt`???
-  //
-  // However, tables.ts specifies:
-  //   NOTATION_FONT_SCALE: 39,
-  //   TABLATURE_FONT_SCALE: 39,
-  const musicFont = Tables.currentMusicFont();
-  const glyph: FontGlyph = musicFont.getGlyphs()[glyphName];
-  const widthInEm = (glyph.xMax - glyph.xMin) / musicFont.getResolution();
-  return widthInEm * 38 * Font.scaleToPxFrom.pt;
+function getGlyphWidth(charCode: number): number {
+  const el = new Element();
+  el.setText(String.fromCharCode(charCode));
+  el.measureText()
```

Comment:
We can discuss this after this PR is merged. Should setText include `this.measureText()` as the last line of its implementation? That way, every time you setText(), the width will be valid. Is there a big performance impact if we do this?

I'm just worried since every time we call setText(...) and change the text, the width is now incorrect. A future developer (or ourselves someday, when we forget this detail), might not know that the width is now incorrect, and can only be fixed by calling measureText() again.

Reply:
There are elements adding padding. I agree that we should look into that. Perhaps an issue to resolve after the migration is complete?